src/tracing_library.cpp

```diff
@@ -178,13 +229,15 @@ std::vector<string_view> TracingLibrary::environment_variable_names() {
           "DD_VERSION"};
 }
 
-string_view TracingLibrary::default_request_operation_name_pattern() { return "nginx.request"; }
+std::string_view TracingLibrary::default_request_operation_name_pattern() {
+  return "nginx.request";
+}
 
-string_view TracingLibrary::default_location_operation_name_pattern() {
+std::string_view TracingLibrary::default_location_operation_name_pattern() {
```

Comment:
Are the values for this variable listed somewhere?

Reply:
Yes: https://github.com/DataDog/nginx-datadog/blob/david.goffredo/dd-trace-cpp/doc/API.md#datadog_proxy_directive